e2etest/bgp_test.go

```diff
@@ -955,3 +1030,18 @@ func uint32Ptr(n uint32) *uint32 {
 func boolPtr(b bool) *bool {
 	return &b
 }
+
+func frrIsCommunitySet(cs clientset.Interface, n *frrcontainer.FRR) {
```

Comment:
I'd have the community fetching logic in this file: https://github.com/metallb/metallb/blob/main/e2etest/pkg/frr/bgp.go
Having a function returning a map community -> refcount.

Reply:
not sure why we need map and ref cnt here

Reply:
As discussed offline, we don't need having a map[community]bool / slice of available communities is enough. RefCnt doesn't matter for our purpouses.